.github/workflows/reusable_provenance.yaml

```diff
@@ -115,23 +125,51 @@ jobs:
         id: ent_upload_binary
         working-directory: downloads
         run: |
-          echo "binary_digest=$(ent put ${{ needs.get_inputs.outputs.artifact-path }})" >> $GITHUB_OUTPUT
+          set -o errexit
+          set -o nounset
+          set -o xtrace
+          set -o pipefail
+          echo "binary_digest=$(ent put --porcelain ${{ needs.get_inputs.outputs.artifact-path }})" >> $GITHUB_OUTPUT
 
       - name: Upload provenance to Ent
         id: ent_upload_provenance
         working-directory: downloads
         # The output on any trigger other than "pull_request" has an addition ".sigstore" suffix.
         # See https://github.com/slsa-framework/slsa-github-generator/tree/main/internal/builders/docker#workflow-outputs
         run: |
-          echo "provenance_digest=$(ent put ${{ needs.get_inputs.outputs.provenance-name }}.sigstore)" >> $GITHUB_OUTPUT
+          set -o errexit
+          set -o nounset
+          set -o xtrace
+          set -o pipefail
+          echo "provenance_digest=$(ent put --porcelain ${{ needs.get_inputs.outputs.provenance-name }}.sigstore)" >> $GITHUB_OUTPUT
```

Comment:
This does not work `pull_request` events. For it to work on `pull_request` events you have to drop `.sigstore` at the end. I am not sure if it is possible to have a conditional here. If not, the condition on line 73 has to change back to `github.event_name != 'pull_request'`.